app/src/main/res/values/dimens.xml

```diff
@@ -1,88 +1,145 @@
+  <dimen name="content_item_question_split_view_margin_bottom">8dp</dimen>
+              <!-- Content Item: Question View -->
+  <dimen name="content_item_question_view_margin_start">32dp</dimen>
+  <dimen name="content_item_question_view_margin_top">32dp</dimen>
+  <dimen name="content_item_question_view_margin_end">32dp</dimen>
+  <dimen name="content_item_question_view_margin_bottom">8dp</dimen>
+              <!-- Feedback Item: Question Split View -->
+  <dimen name="feedback_item_question_split_view_margin_start">32dp</dimen>
+  <dimen name="feedback_item_question_split_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_question_split_view_margin_end">32dp</dimen>
+              <!-- Feedback Item: Question View -->
+  <dimen name="feedback_item_question_view_margin_start">32dp</dimen>
+  <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_question_view_margin_end">32dp</dimen>
+              <!-- Interaction Item: Question View -->
+  <dimen name="interaction_item_question_view_margin_start">32dp</dimen>
+  <dimen name="interaction_item_question_view_margin_top">24dp</dimen>
+  <dimen name="interaction_item_question_view_margin_end">32dp</dimen>
+  <dimen name="interaction_item_question_view_padding_start">32dp</dimen>
+  <dimen name="interaction_item_question_view_padding_end">32dp</dimen>
 
-  <dimen name="clickable_item_min_height">48dp</dimen>
 
-  <dimen name="ongoing_story_card_view_is_tablet_margin_top">28dp</dimen>
-  <dimen name="ongoing_story_card_view_margin_top_portrait_story_grid_greater_then_2">28dp</dimen>
-  <dimen name="ongoing_story_card_view_margin_top_landscape">28dp</dimen>
-  <dimen name="font_scale_content_extra_large_text_view_size">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_end">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_start">28dp</dimen>
+  <!--Drag and Drop-->
+  <dimen name="drag_drop_grouping_hint_split_text_view_margin_top">16dp</dimen>
+  <dimen name="drag_drop_grouping_hint_split_text_view_margin_start">16dp</dimen>
+  <dimen name="drag_drop_grouping_hint_non_split_text_view_margin_top">8dp</dimen>
+  <dimen name="drag_drop_container_parent_non_split_margin_end">8dp</dimen>
+  <dimen name="drag_drop_container_parent_non_split_margin_start">8dp</dimen>
+  <dimen name="drag_drop_content_group_item_margin">8dp</dimen>
+              <!--interaction-->
+  <dimen name="drag_drop_interaction_item_padding_start">16dp</dimen>
+  <dimen name="drag_drop_interaction_item_padding_top">16dp</dimen>
+  <dimen name="drag_drop_interaction_item_padding_end">16dp</dimen>
+  <dimen name="drag_drop_interaction_item_padding_bottom">16dp</dimen>
+  <dimen name="drag_drop_interaction_item_recycler_view_margin_top">4dp</dimen>
+  <dimen name="drag_drop_interaction_items_margin_top">4dp</dimen>
+  <dimen name="drag_drop_interaction_items_margin_bottom">4dp</dimen>
+  
+
+  <!--Profile Chooser-->
```

Comment:
@Yashsomalkar remove the extra space above the comments here also and all other like this.